core/src/main/java/org/elasticsearch/common/unit/SizeValue.java

```diff
@@ -175,9 +175,7 @@ public static SizeValue parseSizeValue(String sValue, SizeValue defaultValue) th
         }
         long singles;
         try {
-            if (sValue.endsWith("b")) {
```

Comment:
We looked at the usage places. Currently only used for specifying thread pool sizes. If a user specified that he wants queues of "5b", then it SHOULD break :-)